### PR TITLE
Improve building docs

### PR DESCRIPTION
**Pull request by rafaeldtinoco**

### CODE REVIEW

**Comment by rafaeldtinoco** [comment]
canceled the testing as they are not needed (And should not even run) for documentation changes.

File: docs/contributing/building/building.md [hunk 1]
```diff
@@ -142,11 +158,11 @@
...
-		" \
-	-v -o dist/tracee-ebpf \
-	./cmd/tracee-ebpf
+ -tags core,ebpf \
+ -ldflags=" \
+  -extldflags \"\" \
+  -X main.version=\"v0.8.0-107-g121efeb\" \
+  " \
+ -v -o dist/tracee-ebpf \
+ ./cmd/tracee-ebpf
```
Comment (geyslan) [inline docs/contributing/building/building.md:167]: It seems broken:

image

**Review by geyslan — CHANGES_REQUESTED** [review]

File: docs/contributing/building/building.md (hunk 1 above)
Comment (rafaeldtinoco) [reply docs/contributing/building/building.md:167]: Hum. I think I know why.. when saving vscode is applying the linting automagically... hum, will fix it.

**Review by geyslan — APPROVED** [review]
LGTM

**Pull request merged**